src/Pirouette/Term/Symbolic/Eval.hs

```diff
@@ -563,9 +562,16 @@ symEvalOneStep t@(R.App hd args) = case hd of
   -- in any other case don't try too hard
   _ -> justEvaluateArgs
   where
```

Comment:
Maybe add a comment here stating that we only evaluate the first argument (if it can be evaluated), instead of evaluating all arguments in parallel as before.